### PR TITLE
Update pattern removal entry in changelog

### PR DESCRIPTION
**Pull request by EoinShaughnessy**
Suggests a few edits to our changelog entry about removing the `pattern` attribute from the date input component.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -53,11 +53,11 @@ This change was introduced in [pull request #2577: Refactor character count to i
 
 #### Remove `pattern` from the date input component
 
-If you're not using the Nunjucks macros, remove `pattern="[0-9]*"` from each of the date input component's elements.
+If you're not using the Nunjucks macros, remove `pattern="[0-9]*"` from each of the date input's elements.
 
-This attribute was originally added to enforce numeric keyboard functionality in Safari on iOS. The `inputmode` attribute provides this functionality for newer Safari versions, so we no longer need to use the `pattern` attribute. 
+We originally added the `pattern`attribute to enforce numeric keyboard functionality in Safari on iOS. However, we no longer need to use it, as the `inputmode` attribute provides this functionality for newer Safari versions.
```
Comment (EoinShaughnessy) [inline CHANGELOG.md:58]: Does 'enforce' have a specific meaning for users, or could we say something like "...to make sure numeric keyboard functionality works in..."?

Comment (EoinShaughnessy) [inline CHANGELOG.md:58]:
```suggestion
We originally added the `pattern` attribute to enforce numeric keyboard functionality in Safari on iOS. However, we no longer need to use it, as the `inputmode` attribute provides this functionality for newer Safari versions.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -53,11 +53,11 @@ This change was introduced in [pull request #2577: Refactor character count to i
 
 #### Remove `pattern` from the date input component
 
-If you're not using the Nunjucks macros, remove `pattern="[0-9]*"` from each of the date input component's elements.
+If you're not using the Nunjucks macros, remove `pattern="[0-9]*"` from each of the date input's elements.
 
-This attribute was originally added to enforce numeric keyboard functionality in Safari on iOS. The `inputmode` attribute provides this functionality for newer Safari versions, so we no longer need to use the `pattern` attribute. 
+We originally added the `pattern` attribute to enforce numeric keyboard functionality in Safari on iOS. However, we no longer need to use it, as the `inputmode` attribute provides this functionality for newer Safari versions.
```
Comment (EoinShaughnessy) [inline CHANGELOG.md:58]: Does 'enforce' have a specific meaning for users, or could we say something like "...to make sure numeric keyboard functionality works in..."?

File: CHANGELOG.md (hunk 1 above)
Comment (querkmachine) [reply CHANGELOG.md]: Probably not either. It's more like it's a "suggestion" that defaulted the on-screen keyboard to a numeric input layout—but the user could still change it to alphanumeric if they wanted to, so it's not really enforced, nor was it functionally necessary for the user to provide valid input.

Comment (EoinShaughnessy) [reply CHANGELOG.md]: Ah cool, thanks for explaining. Could something like the below work?

> We originally added the `pattern` attribute to make numeric keypad functionality the default functionality in Safari on iOS.

Comment (querkmachine) [reply CHANGELOG.md]: Yeah, that makes sense, I think!

Comment (EoinShaughnessy) [reply CHANGELOG.md]: Cheers! :)

File: CHANGELOG.md (hunk 2 above)
Comment (EoinShaughnessy) [inline CHANGELOG.md:58]:
```suggestion
We originally added the `pattern` attribute to make numeric keypad functionality the default functionality in Safari on iOS. However, we no longer need to use it, as the `inputmode` attribute provides this functionality for newer Safari versions.
```

**Review by querkmachine — APPROVED** [review]

**Pull request merged**